helpers.js

```diff
@@ -19,19 +19,572 @@ exports.kodiPlayPause = (request, response) => { // eslint-disable-line no-unuse
+};
+
+//Navigation Left
+exports.kodiNavLeft = (request, response) => { // eslint-disable-line no-unused-vars
+    console.log('Navigate left request received');
+    let Kodi = request.kodi;
+  
+    var navLeft = request.query.q.trim();
+  
+    //Loop for each button press!
+    var i=0;
+    for (i = 0; i < parseInt(navLeft); i++) 
+    { 
+      Kodi.Input.Left();
+    } 
+  
+    response.sendStatus(200);
+};
+
+//Navigation Right
+exports.kodiNavRight = (request, response) => { // eslint-disable-line no-unused-vars
+    console.log('Navigate right request received');
+    let Kodi = request.kodi;
+  
+    var navRight = request.query.q.trim();
+  
+    //Loop for each button press!
+    var i=0;
+    for (i = 0; i < parseInt(navRight); i++) 
+    { 
+      Kodi.Input.Right();
+    } 
+  
+    response.sendStatus(200);
+};
+
+//Navigation Back
+exports.kodiNavBack = (request, response) => { // eslint-disable-line no-unused-vars
+    console.log('Navigate back request received');
+    let Kodi = request.kodi;
+  
+    var navBack = request.query.q.trim();
+  
+    //Loop for each button press!
+    var i=0;
+    for (i = 0; i < parseInt(navBack); i++) 
+    { 
+      Kodi.Input.Back();
+    } 
+  
+    response.sendStatus(200);
+};
+
+//Navigation Select
+exports.kodiNavSelect = (request, response) => { // eslint-disable-line no-unused-vars
+    console.log('Navigation select request received');
+    let Kodi = request.kodi;
+  
+    Kodi.Input.Select();
+    response.sendStatus(200);
+};
+
+//Navigation ContextMenu
+exports.kodiNavContextMenu = (request, response) => { // eslint-disable-line no-unused-vars
+    console.log('Navigation ContextMenu request received');
+    let Kodi = request.kodi;
+  
+    Kodi.Input.ContextMenu();
+    response.sendStatus(200);
+};
+
+// Show Info
+exports.kodiDisplayInfo = (request, response) => { // eslint-disable-line no-unused-vars
+    console.log('Display information request received');
+    let Kodi = request.kodi;
+  
+    Kodi.Input.Info();
+    response.sendStatus(200);
+};
+
+//Navigation Home
+exports.kodiNavHome = (request, response) => { // eslint-disable-line no-unused-vars
+    console.log('Navigation Home request received');
+    let Kodi = request.kodi;
+  
+    Kodi.Input.Home();
+    response.sendStatus(200);
+};
+
+// Set subtitles
+exports.kodiSetSubs = (request, response) => { // eslint-disable-line no-unused-vars
+    let Kodi = request.kodi;
+  
+    var setsubs = request.query.q.trim();
+    console.log('Change subtitles request received - ' + setsubs);
+
+    if(setsubs == "previous" || setsubs == "next" || setsubs == "on" || setsubs == "off")
+    {      
+      Kodi.Player.SetSubtitle({"playerid":1,"subtitle":setsubs});
+	  
+	  console.log('Change subtitle request received - ' + setsubs);
+    }
+    else
+    {
+     //If not on/off/previous/next then set the subtitles to an index number
+      var subtitleStreamIndex;
+      
+      if(setsubs == "zero")
```

Comment:
These if , else if, else if.. etc. Can be changed to a mapping. You can use an object for that.
```
const numberMapping = new map([["zero", 0], ["one", 1], ["two", 2], ... ])
```
Also this can be reused, so better to create a utility function for this.

More info here: https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Map